api/storage-providers.ts

```diff
@@ -0,0 +1,9 @@
+import { client } from "./client.ts";
+
+export const storageProviders = {
+  create: client("/storage").post,
+  get: client("/storage/{id}").get,
+  list: client("/storage").get,
+  update: client("/storage/{id}").put,
+  delete: client("/storage/{id}").delete,
```

Comment:
shouldn't it be `storages`?  like snapshots, etc. ?
I see in api the path is storage, but I think we should keep it consistent with the rest of the routers. I dont see ps-web using it yet, maybe is a quick fix?